session_items.py

```diff
@@ -1,77 +1,47 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+import requests
+from trello_cards import make_trello_auth, get_cards_url_with_auth
+import os
 
 def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS)
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-
-def add_item(title):
-    """
-    Adds a new item with the specified title to the session.
-
-    Args:
-        title: The title of the item.
-
-    Returns:
-        item: The saved item.
-    """
-    items = get_items()
+    """ Simple attempt to get all cards from Trello. """
+    response = requests.get(make_trello_auth(f"https://api.trello.com/1/boards/{os.getenv('BOARD_ID')}/cards"))
+    todos = response.json()
+    for card in todos:
+        print(card ['name'], card['desc'])        
+    return todos
 
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
 
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
+def get_done_items():
+    """ Simple attempt to get all done cards from Trello. """
+    response = requests.get(f"https://api.trello.com/1/lists/{os.getenv('DONE_idList')}/cards?key={os.getenv('TRELLO_API_KEY')}&token={os.getenv('TRELLO_API_TOKEN')}")
```

Comment:
You can pass the query parameters as a dictionary:
```suggestion
    params = { "key":  os.getenv('TRELLO_API_KEY'), "token": os.getenv('TRELLO_API_TOKEN')}
    response = requests.get(f"https://api.trello.com/1/lists/{os.getenv('DONE_idList')}/cards", params = params)
```